pandas/core/groupby/generic.py

```diff
@@ -240,6 +240,7 @@ def apply(self, func, *args, **kwargs) -> Series:
         Returns
         -------
         Series or DataFrame
+            The resulting dtype will reflect the return value of the passed ``func``.
```

Comment:
This doesn't seem very clear to me. Why not something like `A pandas object with the result of applying ``func`` to each group.`. Or maybe you can think of something better, but the return description should focus on what's being returned, not which type is being returned.

Reply:
Make sense. Couldn't think of anything else so I use yours.